ee/desktop/user/client/dialer_posix.go

```diff
@@ -9,7 +9,8 @@ import (
 )
 
 func dialContext(socketPath string) func(_ context.Context, _, _ string) (net.Conn, error) {
-	return func(_ context.Context, _, _ string) (net.Conn, error) {
-		return net.Dial("unix", socketPath)
+	return func(ctx context.Context, _, _ string) (net.Conn, error) {
+		d := net.Dialer{}
+		return d.DialContext(ctx, "unix", socketPath)
```

Comment:
wonder if we should reuse dialer?

```suggestion
        d := net.Dialer{}
	return func(ctx context.Context, _, _ string) (net.Conn, error) {
		return d.DialContext(ctx, "unix", socketPath)
```

Reply:
updated to do the same thing the net pkg does